imblearn/over_sampling/smote.py

```diff
@@ -41,6 +42,10 @@ class SMOTE(BaseBinarySampler):
 
     k_neighbors : int, optional (default=5)
```

Comment:
I would say:
k_neighbors : int or object, optional (default=5)
If int, the number of the nearest neighbours to be used to construct synthetic samples.
If object, an estimator that inherits from sklearn.neighbors.base.KNeighborsMixin